content/200-orm/100-prisma-schema/10-overview/03-generators.mdx

```diff
@@ -55,6 +59,125 @@ If you use macOS ARM64 (`darwin-arm64`), then the binary file that was compiled
+
+```txt
+.
+├── package.json
+├── prisma
+│   └── schema.prisma
+├── src
+│   └── index.ts
+└── tsconfig.json
+```
+
+Then `../src/generated/prisma/client` places the generated code in `src/generated/prisma/client` relative to `schema.prisma`.
+
+#### 2. Generate the Prisma Client
+
+Generate the `PrismaClient` by running:
+
+```bash
+npx prisma generate
+```
+
+This compiles the necessary TypeScript definitions and includes the query engine in the configured `output` folder.
+
+#### 3. Exclude the generated directory from version control
+
+The new generator includes both the TypeScript client code **and** the [query engine](/orm/more/under-the-hood/engines#the-query-engine-file). Including the query engine in version control can cause compatibility issues on different machines. To avoid this, add the generated directory to `.gitignore`:
+
+```bash file=".gitignore"
+# Recommended:
+/src/generated/prisma/client
+```
+
+:::note
+
+In the future, you can safely include the generated directory in version control when [Prisma ORM is fully transitioned from Rust to TypeScript](https://www.prisma.io/blog/rust-to-typescript-update-boosting-prisma-orm-performance?utm_source=docs&utm_medium=inline_text).
+
+:::
+
+#### 4. Use the Generated Client in Your Application
+
+After generating the Prisma Client, import the types from the path you specified:
+
+```ts file="src/index.ts"
+import { PrismaClient } from "./generated/prisma/client"
+
+const prisma = new PrismaClient()
+```
+
+Your Prisma Client is now ready to use in your Node.js project.
+
+### Generator options reference
+
+Use the following options in the `generator client { ... }` block. Only `output` is required. The other fields have defaults or are inferred from your environment and `tsconfig.json`.
+
+```prisma file="schema.prisma"
+generator client {
+  // Required
+  provider = "prisma-client"
+  output   = "../src/generated/prisma/client"
+
+  // Optional
+  runtime                = "nodejs"
+  moduleFormat           = "esm"
+  generatedFileExtension = "ts"
+  importFileExtension    = "ts"
+}
+```
+
+Below are the options for the `prisma-client` generator:
+
+
+| **Option**                | **Default**                  | **Description**                                                                                                                                          |
+|---------------------------|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `provider`               | `prisma-client-js`                          | It can be `prisma-client-js` or `prisma-client`.                                                                                                                          |
```

Comment:
this line isn't needed since the entire section is about the `prisma-client` generator

```suggestion
```